sql/core/src/test/scala/org/apache/spark/sql/FilterPushdownBenchmark.scala

```diff
@@ -0,0 +1,230 @@
+    Parquet Vectorized                            4321 / 5021          0.2        4121.3       1.0X
+    Parquet Vectorized (Pushdown)                 3967 / 4183          0.3        3783.6       1.1X
+    Native ORC Vectorized                         4107 / 4565          0.3        3916.9       1.1X
+    Native ORC Vectorized (Pushdown)              2983 / 3861          0.4        2844.5       1.4X
+
+    Select 90% rows (id < 943718):          Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+    -----------------------------------------------------------------------------------------------
+    Parquet Vectorized                            6815 / 7287          0.2        6499.0       1.0X
+    Parquet Vectorized (Pushdown)                 6891 / 7220          0.2        6571.5       1.0X
+    Native ORC Vectorized                         7337 / 7565          0.1        6997.1       0.9X
+    Native ORC Vectorized (Pushdown)              7274 / 7523          0.1        6936.6       0.9X
+
+    Select all rows (id IS NOT NULL):       Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+    -----------------------------------------------------------------------------------------------
+    Parquet Vectorized                            7321 / 7380          0.1        6981.5       1.0X
+    Parquet Vectorized (Pushdown)                 7352 / 7398          0.1        7011.2       1.0X
+    Native ORC Vectorized                         7386 / 7660          0.1        7043.9       1.0X
+    Native ORC Vectorized (Pushdown)              7629 / 7705          0.1        7275.9       1.0X
+
+    Select all rows (id > -1):              Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+    -----------------------------------------------------------------------------------------------
+    Parquet Vectorized                            7125 / 7384          0.1        6795.2       1.0X
+    Parquet Vectorized (Pushdown)                 7334 / 7390          0.1        6994.3       1.0X
+    Native ORC Vectorized                         7517 / 7642          0.1        7168.7       0.9X
+    Native ORC Vectorized (Pushdown)              7323 / 7601          0.1        6983.7       1.0X
+
+    Select all rows (id != -1):             Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+    -----------------------------------------------------------------------------------------------
+    Parquet Vectorized                            7281 / 7850          0.1        6944.0       1.0X
+    Parquet Vectorized (Pushdown)                 7311 / 7939          0.1        6972.7       1.0X
+    Native ORC Vectorized                         7530 / 7748          0.1        7181.4       1.0X
+    Native ORC Vectorized (Pushdown)              7309 / 7667          0.1        6970.2       1.0X
+    */
+    benchmark.run()
+  }
+
+  def main(args: Array[String]): Unit = {
+    val numRows = 1024 * 1024
+    val width = 20
```

Comment:
I think `20` is not a common table width, how about `5`?

Reply:
No problem. I'll set to 5.